content/docs/pulumi-cloud/deployments/reference.md

```diff
@@ -223,6 +223,19 @@ These can be overridden or extended by configuring custom environment variables:
 
 ![Pulumi UI - Environment Variables](../ui-custom-env-variables.png)
 
+#### PULUMI_ENV
+
+Environment variables can be persisted between pre-run commands and the final pulumi deployment by appending them to the file on the file system named `PULUMI_ENV`.
+
+Example Usage:
+
+```bash
+export GOOGLE_OAUTH_ACCESS_TOKEN=$(gcloud auth print-access-token)
+echo GOOGLE_OAUTH_ACCESS_TOKEN=$GOOGLE_OAUTH_ACCESS_TOKEN >> /pulumi/projects/PULUMI_ENV
```

Comment:
Any context on why this filepath is `/pulumi/projects/PULUMI_ENV` - I fear this wouldn't be clear to a new user.

Reply:
It looks like its weird on the image I had running. Logs from app.pulumi.com has it at /, `Loading PULUMI_ENV from "/"`